daemon/graphdriver/overlay2/overlay.go

```diff
@@ -213,7 +216,12 @@ func parseOptions(options []string) (*overlayOptions, error) {
 			if err != nil {
 				return nil, err
 			}
-
+		case "overlay2.default_size":
```

Comment:
So you removed the "default_" part and not the "overlay2." part. Oh well, I guess there are arguments to justify both alternatives. Either are fine by me.